core/src/main/java/org/opensearch/sql/analysis/ExpressionAnalyzer.java

```diff
@@ -191,9 +192,9 @@ public Expression visitWindowFunction(WindowFunction node, AnalysisContext conte
   }
 
   @Override
-  public Expression visitHighlight(HighlightFunction node, AnalysisContext context) {
-    Expression expr = node.getHighlightField().accept(this, context);
-    return new HighlightExpression(expr);
+  public Expression visitHighlightFunction(HighlightFunction node, AnalysisContext context) {
+    return (node.getName() != null) ? new ReferenceExpression(node.getName(), ExprCoreType.STRING)
+        : null;
```

Comment:
Why is this `null` instead of `ExprNullValue.of()` or `nullValue()`

Reply:
Because we need a return type of `Expression` rather than `ExprValue`.